example/package.json

```diff
@@ -40,7 +40,7 @@
     "example:scan-path-ledger": "node -r ts-node/register --preserve-symlinks scan-path-ledger.ts"
   },
   "dependencies": {
-    "@ledgerhq/hw-transport-node-hid": "^6.20.0",
+    "@ledgerhq/hw-transport-node-hid": "^6.27.1",
```

Comment:
are the examples running without errors as well?

Reply:
i couldnt see any problems but i can go throught them one at a time